lang/en-gb.xml

```diff
@@ -284,10 +331,40 @@ Do not power off the system or close the application.</warning_saving>
 		<completed_setup>You have now completed initial setup.
 Your Vita3K system is ready!</completed_setup>
 		<select_language>Select a language.</select_language>
+		<select_pref_path>Select a pref path.</select_pref_path>
+		<current_emu_path>Current emulator path</current_emu_path>
+		<change_emu_path>Change Emulator Path</change_emu_path>
+		<reset_emu_path>Reset Emulator Path</reset_emu_path>
+		<install_fw>Install Firmware.</install_fw>
+		<install_highly_recommended>Installing both firmware files is highly recommended.</install_highly_recommended>
+		<installed>Installed:</installed>
+		<download_firmware>Download Firmware</download_firmware>
+		<download_font_package>Download Font Package</download_font_package>
+		<install_fw_file>Install Firmware File</install_fw_file>
+		<select_interface_settings>Select interface settings.</select_interface_settings>
+		<info_bar_visible>Info Bar Visible</info_bar_visible>
+		<info_bar_visible_note>Check the box to show info bar inside app selector.
+Info bar is clock, battery level and notification center.</info_bar_visible_note>
+		<livearea_app_screen>Live Area App Screen</livearea_app_screen>
+		<livearea_app_screen_note>Check the box to open Live Area by default when clicking on an application.
+If disabled, right click on an application to open it.</livearea_app_screen_note>
+		<grid_mode>Grid Mode</grid_mode>
+		<grid_mode_note>Check the box to set the app list to grid mode like of PS Vita.</grid_mode_note>
+		<app_icon_size>App Icon Size</app_icon_size>
+		<select_icon_size>Select your preferred icon size.</select_icon_size>
+		<completed>Completed.</completed>
```

Comment:
revert all of this

Reply:
i want keep the "completed", so that know the initial setup is complete.

Reply:
That no already exist ?

Reply:
> That no already exist ?

other strings are revert, just keep the "completed".

Reply:
Ah ok, justr this text no exist on vita i have jyst added it like that but ok 👌 keep it